.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,5 @@
+blank_issues_enabled: false
+contact_links:
+  - name: Discord
+    url: https://discord.com/invite/dfYAWVMaCW
+    about: Ask questions and discuss with other Pyrgg community members
```

Comment:
PyRGG

Reply:
fixed in [d9c85ff](https://github.com/sepandhaghighi/pyrgg/pull/161/commits/d9c85fff6d627482a572ea7afcc5d2dd42377a84).